profiler/kokoro/integration_test.sh

```diff
@@ -34,14 +34,29 @@ export GOOGLE_APPLICATION_CREDENTIALS="${KOKORO_KEYSTORE_DIR}/72935_cloud-profil
 export GCLOUD_TESTS_GOLANG_PROJECT_ID="cloud-profiler-e2e"
 
 # Ensure a newer version of Go is used so it is compatible with newer libraries.
-# Here we install v1.17.7 which is the current version as of when this code
-# was written, following instructions from https://go.dev/doc/manage-install.
-# Go modules might not be on for previous versions of Go, so we also have to
-# enable the module system explicitly.
-export GO111MODULE=on
-go install golang.org/dl/go1.17.7
-go1.17.7 download
+# Here we install v1.18.4 which is the current version as of when this code
+# was written.
```

Comment:
nit: which is the latest version as of July 2022.

Reply:
Done.